.github/workflows/ci.yaml

```
@@ -684,6 +684,10 @@ jobs:
   send-notification:
     needs:
       - diff-dumps
+      - upload-db-dump
+      - upload-dumps-for-downstream
+      - upload-dumps-for-embedding
+      - push-manifests
     runs-on: ubuntu-latest
     if: failure()
```

Do we want to be notified if these jobs fail at _any_ point, e.g., including PRs? If not, we might want to add additional checks (not 100% sure if this is correct):
```suggestion
    if: failure() && (github.ref_name == 'master' || startsWith(github.ref, 'refs/tags/'))
```